airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorMetadataDefinitionV0.py

```diff
@@ -118,6 +118,14 @@ class Config:
     ql: Optional[Literal[100, 200, 300, 400, 500, 600]] = None
 
 
+class PyPi(BaseModel):
+    class Config:
+        extra = Extra.forbid
+
+    enabled: bool
+    packageName: str = Field(..., description="The name of the package on PyPi.")
```

Comment:
For a followup PR once the publish to PyPi step is implemented:
* We should add a validator that makes sure that if `enabled is True` the current connector version is available on PyPi.
It's basically what we do for docker images: before uploading the metadata file to GCS we validate that the docker image is available on DockerHub.

Reply:
you mean as a post-upload validator? If yes that makes a ton of sense to me, thanks for mentioning

Reply:
Yes when we run `metadata_service validate <config file path>`

Reply:
In this module: https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/metadata_service/lib/metadata_service/validators/metadata_validator.py#L1